GIBSDownloader/tile_utils.py

```diff
@@ -198,15 +244,32 @@ def img_to_tiles(cls, tiff_path, region, res, tile, tile_date_path, img_format,
             print("Generating {} tiles sequentially...".format(len(pixel_coords)), flush=True)
 
         if ultra_large: 
-            TileUtils.generate_ultra_large_tiles(tiff_path, tile, WIDTH, HEIGHT, pixel_coords, img_format, mp, metadata.date)
+            TileUtils.generate_ultra_large_tiles(originals_path, tile, WIDTH, HEIGHT, pixel_coords, img_format, mp, metadata.date)
         else: 
-            TileUtils.generate_regular_tiles(tiff_path, tile, WIDTH, HEIGHT, pixel_coords, img_format, mp)
+            TileUtils.generate_regular_tiles(originals_path, tile, WIDTH, HEIGHT, pixel_coords, img_format, mp)
         print("done!")
 
     @classmethod
     def generate_pixel_coords(cls, tile, WIDTH, HEIGHT, num_rows, num_cols, metadata, geotran_d, tile_date_path, mp):
         """
-            @returns a list with all pixel coordinate information for the tiles that will be generated
+        Helper which returns a list with all pixel coordinate information for 
+        the tiles that will be generated.
+
+        Parameters:
+            tile (Tile): Tile object storing tiling information
+            WIDTH (int): width of the original image
```

Comment:
Any reason why width and height are capitalised?

Reply:
I left them capitalized to differentiate between widths and heights of tiles, which are non-constant and may change throughout the program's execution. Once these values are assigned, (corresponding to the original downloaded image's width and height), they are no longer changed, and so I thought it appropriate to have them capitalized.